static/js/containers/LearnerPage.js

```diff
@@ -95,11 +97,16 @@ class LearnerPage extends React.Component<*, LearnerPageProps, *> {
         ...profileProps(profile)
       };
       toRender = childrenWithProps(children, props);
+      name = `${profile.profile.preferred_name} ${profile.profile.last_name}'s`;
     }
     const { errorInfo } = profile;
-    return <Loader loaded={loaded}>
-      {errorInfo && loaded ? <ErrorMessage errorInfo={errorInfo} /> : toRender }
-    </Loader>;
+    return (
+      <DocumentTitle title={`${name} MicroMasters Profile`}>
```

Comment:
If I do a full refresh on this page (`/learner`) I see `undefined undefined MicroMasters Profile` for a second while the request for the profile data is in-flight.

I think we should take a different approach - maybe we should have a little helper function that formats the title, something like:

```js
getDocumentTitle = () => {
  const {
    params: { username },
    profiles,
  } = this.props;
  
  if ( profiles[username] !== undefined ) {
    let first = R.propOr('', 'preferred_name', profiles[username]);
    let last = R.propOr('', 'last_name', profiles[username]);
    return `${first} ${last} MicroMasters Profile`.trim();
  }
  return 'MicroMasters Profile';
}
```

probably there's a more elegant way to express that but you get the idea. We don't ever want to stick `undefined` into a template, because `${undefined}` evaluates to the string `"undefined"`.